modules/apps/object/object-api/src/main/java/com/liferay/object/util/LanguageUtils.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.object.util;
+
+import com.liferay.portal.kernel.util.LocaleUtil;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * @author Matheus Almeida
+ */
+public class LanguageUtils {
+
+	public static Map<String, String> getLanguageIdMap(
```

Comment:
Hey @mlan0 ,

Would you mind to move this method to the class com.liferay.object.util.LocalizedMapUtil?
Another option is take a look on this class (com.liferay.portal.vulcan.util.LocalizedMapUtil) checking if some existing method can help you with. 

Thanks!